apps/frontend/src/network/users/groups/__tests__/GroupsCard.test.tsx

```diff
@@ -63,7 +63,7 @@ it('is rendered when there are groups', async () => {
   );
   await waitFor(() => {
     expect(queryByText(/loading/i)).not.toBeInTheDocument();
-    expect(queryByText(/test’ groups/i)).toBeInTheDocument();
+    expect(queryByText(/test’s groups/i)).toBeInTheDocument();
```

Comment:
Yeah I guess this is overly specific.
Something like this may work, you'll also have to change in at least one other place in the file up there :point_up: 
```suggestion
    expect(queryAllByText(/groups/i)).toBeInTheDocument();
```